qdrant-landing/content/documentation/guides/optimize.md

```diff
@@ -259,6 +276,23 @@ client
     .get();
 ```
 
+```csharp
+using Qdrant.Client;
+using Qdrant.Client.Grpc;
+
+var client = new QdrantClient("localhost", 6334);
+
+await client.SearchAsync(
+	"{collection_name}",
+	new float[] { 0.2f, 0.1f, 0.9f, 0.7f },
```

Comment:
I propose to add names to other arguments

Reply:
https://github.com/qdrant/landing_page/pull/607/commits/d14b76c88a046f08c2bd47abef6f98a437c1ec15